### PR TITLE
#765 remove empty css files

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Removed Empty CSS files

## Test Cases

Smoke test to make sure that nothing in the application is broken/ looks weird

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors
- [x] No newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (if applicable)
- [x] Remove any not-applicable sections
- [x] Assign the PR to yourself
- [x] PR is linked to the ticket
- [x] No `package-lock.json` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack

Closes # (issue #)

### CODE REVIEW

File: src/frontend/pages/ProjectsPage/projects-table/projects-table.module.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /*
  * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
  * See the LICENSE file in the repository root folder for details.
- */
+ */
```
Comment (jamescd18) [inline src/frontend/pages/ProjectsPage/projects-table/projects-table.module.css:4]: Delete this file too?

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Is this everything? Just one file that looks like it should have been deleted but was just edited?

File: src/frontend/pages/ProjectsPage/projects-table/projects-table.module.css (hunk 1 above)
Comment (anthonybernardi) [reply src/frontend/pages/ProjectsPage/projects-table/projects-table.module.css]: 100% go for it

**Comment by anthonybernardi** [comment]
Nothing better than a PR that looks like this: 

Screen Shot 2022-07-18 at 9 39 57 PM

**Review by jamescd18 — APPROVED** [review]
LGTM!

**Comment by RChandler234** [comment]
I couldn't move all the css over, some was too complex to put into a jsx object; I suggest we make a stylesheets folder for the frontend and move the remaining ones there for now and try to find a more consistent way to handle having css in a tsx file (there are ways, but it'd involved adding more packages/ bigger changes, so I don't wanna do it rn); that or if this way looks too inconsistent lmk, and we can just do removing the unused stylesheets for this MR and handle moving the CSS in another one. @jamescd18 @anthonybernardi

**Comment by jamescd18** [comment]
I see nothing wrong with only have a few components have CSS files, I don't think there's any need to make them be in a separate folder.

**Review by anthonybernardi — APPROVED** [review]
i'm good with this. we can deal with the last few files in a separate ticket. I wouldn't mind a stylesheets folder actually since my plan is to move all the tests out too

**Pull request merged**